androidTest/java/com/owncloud/android/authentication/AuthenticatorActivityTest.java

```diff
@@ -206,12 +206,12 @@ public void test2_check_certif_not_secure()
 
             // Check that login button is disabled
```

Comment:
`// Check that login button is not displayed`

Reply:
Done!